components/deck/build-deck.webc

```diff
@@ -226,3 +226,144 @@
   }
 }
 </style>
+
+<style webc:bucket="slides-theme">
+  @layer slide.theme {
+
+    slide-deck {
+
+      /* background colors */
+      --slide-background: hsl(200 12% 95.098%);
+      --slide-callout-color: hsl(193deg 28% 87%);
+      --slide-mark-bg-color: var(--slide-callout-color);
+
+      /* content colors */
+      --slide-text-color: hsl(200deg 12% 4.902%);
+      --slide-text-light-color: hsl(192.86deg 10% 27.451%);
+      --slide-accent-color: hsl(330.2deg 67.742% 42.549%);
+      --slide-border-color: hsl(195deg 10% 50%);
+      --slide-shadow-color: hsl(195deg 30.769% 94.902% / 0.4);
+      --slide-action-color: hsl(195deg 53% 27%);
+      --slide-active-color: hsl(330deg 52.857% 27.451%);
+      --slide-highlight-color: hsl(330deg 52.857% 27.451%);
+      --slide-feature-color: hsl(195.65deg 45.098% 20%);
+      --slide-strikeout-color: var(--slide-highlight-color);
+  
+      /* fonts */
+      --slide-os-serif-font: palatino, 'palatino linotype', 'palatino lt std', 'book antiqua',
+        georgia;
+      --slide-os-sans-font: 'helvetica neue', helvetica, arial;
+      --slide-os-code-font: sfmono-regular, consolas, monaco;
+
+      --slide-serif-font: var(--slide-web-font-serif, var(--slide-os-serif, ui-serif)), serif;
+      --slide-sans-serif-font: var(--slide-web-font-sans, var(--slide-os-sans-font, ui-sans-serif)), sans-serif;
+      --slide-code-font: var(--slide-web-font-code, var(--slide-os-code-font, ui-monospace)), monospace, serif;
+
+      background: var(--slide-background);
+      color: var(--slide-text-color);
+      color-scheme: light;
+    }
+
+    body {
```

Comment:
We should maybe apply these to slide-deck instead, so we're only touching styles inside the component? We could also scope the styles using `webc:scope` if we really want to avoid outside conflicts.

Reply:
I moved all of the theme styles inside of slide-deck. Would we have to set up the `slide-deck` component in a different way to apply `webc:scope`? When I add that to the theme style tag now a lot of the styles look broken.

Reply:
Yeah, we're styling stuff here that should apply inside nested components, and I don't think webc:scope handles that. So maybe best not to for now. We can revisit at some point.